### ISSUE TITLE
Lost UI shared context.

### ISSUE THREAD

**Issue by noraluk-scg** [davidteather/TikTok-Api#297]
```
DevTools listening on ws://127.0.0.1:45037/devtools/browser/a68cb9d1-c4c7-4480-aa66-620c40e2fdd7

[1024/074722.607648:INFO:CONSOLE(1)] "%c", source: (1)

[1024/074723.928035:ERROR:gpu_process_transport_factory.cc(980)] Lost UI shared context.
```

**Comment by jjaacckk** [comment]
yeah, I just found the same error. i'm running python 3.8.2 on ubuntu 20.04.1

**Comment by elblogbruno** [comment]
also I can't play playAddrs url  on latest version when getting trending tiktoks 

![image](https://user-images.githubusercontent.com/10481058/97083707-4eab3e80-1612-11eb-93f9-1cbcba515bbe.png)

**Comment by davidteather** [comment]
I did end up making a change for browser and am reverting it in my current PR.

@elblogbruno see #296 this means if your chrome updates at all you won't be able to open it in your browser. You can use other methods of the API to download the video locally.

**Comment by davidteather** [comment]
Does the latest version solve your issue

**Comment by elblogbruno** [comment]
> I did end up making a change for browser and am reverting it in my current PR.
> 
> @elblogbruno see #296 this means if your chrome updates at all you won't be able to open it in your browser. You can use other methods of the API to download the video locally.

ok thanks i will see how i update my code

**Comment by davidteather** [comment]
Most likely fixed in 3.5.7 closing due to no activity
